src/modm/platform/clock/stm32/module.lb

```diff
@@ -105,6 +105,16 @@ def build(env):
     else:
         properties["cfgr2"] = "CFGR"
 
+    if target.family in ["h7"]:
+        if target.name in ["a0", "a3", "b0", "b3"]:
```

Comment:
```suggestion
        if target.name in ["a3", "b0", "b3"]:
```

There are no H7A0 devices

Reply:
Manually applied this suggestion and fixed the other instances of the same. I was wondering what was going on here 🙂